carina-core/src/main/java/com/qaprosoft/carina/core/foundation/listeners/CarinaListener.java

```diff
@@ -338,7 +338,7 @@ public void onFinish(ITestContext context) {
 
         // [VD] seems like useless after movemevt driver quite onto afterMethod
```

Comment:
please replace this comment with message that quitD... must present and can't be removed and reason

Reply:
done @vdelendik